client/state/data-layer/wpcom/me/blocks/sites/index.js

```diff
@@ -0,0 +1,46 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { READER_SITE_BLOCKS_REQUEST, READER_SITE_BLOCKS_RECEIVE } from 'state/action-types';
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { registerHandlers } from 'state/data-layer/handler-registry';
+
+export const handleSiteBlocksRequest = action =>
+	http(
+		{
+			apiNamespace: 'wpcom/v2',
+			method: 'GET',
+			path: '/me/blocks/sites',
+			query: {
+				page: action.page || null,
+				per_page: action.perPage || null,
+			},
+		},
+		action
+	);
+
+export const siteBlocksRequestReceived = ( action, apiResponse ) => ( {
+	type: READER_SITE_BLOCKS_RECEIVE,
+	payload: apiResponse,
+} );
+
+export const siteBlocksRequestFailure = error => ( {
+	type: READER_SITE_BLOCKS_RECEIVE,
+	payload: error,
```

Comment:
Do we need a separate `payload` and `error: true`? I think `error` is unnecessary here. Also FWIW, I'd rename `payload` to `error,` but that's my personal preference and I don't have strong feelings on that.

Reply:
I think Chris is following flux-standard-action here https://github.com/redux-utilities/flux-standard-action